tests/conftest.py

```
@@ -0,0 +1,64 @@
+    project_root = Path(__file__).parent.parent
+    app_src = project_root / "app" / "src"
+    if str(app_src) not in sys.path:
+        sys.path.insert(0, str(app_src))
+@pytest.fixture
+def project_root() -> Path:
+    """Return the project root directory."""
+    return Path(__file__).parent.parent
+
+
+@pytest.fixture
+def project_root() -> Path:
```

Ruff (0.14.3)

Redefinition of unused `project_root` from line 16

(F811)